terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,69 @@
+# Define all hardcoded local variable and local variables looked up from data resources
+locals {
+  stack_name                  = "company-requests" # this must match the stack name the service deploys into
+  name_prefix                 = "${local.stack_name}-${var.environment}"
+  global_prefix               = "global-${var.environment}"
+  service_name                = "customer-feedback-api"
+  container_port              = "8080"
+  docker_repo                 = "customer-feedback-api"
+  kms_alias                   = "alias/${var.aws_profile}/environment-services-kms"
+  lb_listener_rule_priority   = 31
```

Comment:
This priority is already in use

Reply:
How should it be determined?

Reply:
At the moment we don't have designated priorities for api services but it we could either have a lookup list for each service or have a location where all the priorities are updated. I just know this is in use because I wrote the template